lemur/certificates/models.py

```diff
@@ -454,6 +454,18 @@ def update_destinations(target, value, initiator):
 
     if target.expired:
         return
+
+    if current_app.config.get("SYNCHRONOUS_DESTINATION_UPLOAD", True) == False:
```

Comment:
@jonas-p I think it makes more sense to have a default of True instead of using the fact that `None == False`

Reply:
`None == False` is `False` and would mean that not having `SYNCHRONOUS_DESTINATION_UPLOAD` in the configuration would mean the same as setting it to `False` which is not intuitive